app-emulation/docker/docker-9999.ebuild

```diff
@@ -260,16 +249,13 @@ src_compile() {
 
 	# time to build!
 	./hack/make.sh dynbinary || die 'dynbinary failed'
-
-	# build the man pages too
-	#./man/md2man-all.sh || die "unable to generate man pages"
 }
 
 src_install() {
 	VERSION="$(cat VERSION)"
 	newbin "bundles/$VERSION/dynbinary/docker-$VERSION" docker
-	exeinto /usr/libexec/docker
-	newexe "bundles/$VERSION/dynbinary/dockerinit-$VERSION" dockerinit
+	#exeinto /usr/libexec/docker
+	#newexe "bundles/$VERSION/dynbinary/dockerinit-$VERSION" dockerinit
```

Comment:
I'm guessing dockerinit is replaced by runc magic? Prune the old lines, not comment them.